### DISCUSSION TITLE
TURKISH LANGUAGE MUST BE ADDED <3

### DISCUSSION THREAD

**Discussion by crosproata** [IBAX-io/go-ibax#194] [Ideas] [answered]
There is so many turkish followers for this coin and wallet and some of them dont know english,this project is a big project,i think turkish community is a big fan and helper.

**Comment by ibaxio** [comment, marked as answer]
The IBAX team is recruiting volunteers to translate the official website. Due to the rapid growth of the Turkish community, Turkish will be uploaded in the next revision.

We also welcome translations from community members in other languages.

Unleash the power of our community and make us stronger.
